app/views/device_stories/show.html.erb

```diff
@@ -47,5 +49,40 @@
   </div>
 </div>
 
+<div>
+  <h2>Comments: <br></h2>
+  <table class="table table-striped">
+    <tbody>
+    <%- if @device_story.device_story_comments.empty? %>
+      No Comments yet.
+    <%- else %>
+      <%- @device_story.device_story_comments.each do |comment| %>
+        <tr>
+          <td>
+            <b><%= link_to comment.user.name , comment.user %></b> <div style="display: inline-block; color: dimgrey; font-size: small">(<%= time_ago_in_words(comment.created_at) %> ago)</div> <br>
+            <%= comment.content %>
+            <%- if user_signed_in? -%>
+              <%- if (comment.user_id == current_user.id) || moderator?(current_user) %>
```

Comment:
Looks like you got the right idea here, but we should protect the route as well. People could build their own requests to delete the user if the API allows it.